src/Client/Shared/NavMenu.razor

```diff
@@ -115,5 +120,8 @@
         _canViewChat = (await _authorizationService.AuthorizeAsync(_authenticationStateProviderUser, Permissions.Communication.Chat)).Succeeded;
         _canViewProducts = (await _authorizationService.AuthorizeAsync(_authenticationStateProviderUser, Permissions.Products.View)).Succeeded;
         _canViewBrands = (await _authorizationService.AuthorizeAsync(_authenticationStateProviderUser, Permissions.Brands.View)).Succeeded;
+
+        _accessToken = await _localStorage.GetItemAsync<string>(StorageConstants.Local.AuthToken);
+        _jobsLink = $"/jobs?token={_accessToken}";
```

Comment:
Is it safe to pass a token in a query string?

Reply:
The token is exposed in the header of any call, so is more or less the same as exposing it in the query string.